src/os/ui/file/kml/abstractkmlexporter.js

```diff
@@ -712,10 +712,11 @@ os.ui.file.kml.AbstractKMLExporter.prototype.addGeometryNode = function(item, no
  * @param {T} item The item
  * @param {string} styleId The style id
  * @param {string} color The item color
+ * @param {string} fillColor The item fill color
  * @param {os.ui.file.kml.Icon=} opt_icon The item icon
  * @protected
  */
-os.ui.file.kml.AbstractKMLExporter.prototype.createStyle = function(item, styleId, color, opt_icon) {
+os.ui.file.kml.AbstractKMLExporter.prototype.createStyle = function(item, styleId, color, fillColor, opt_icon) {
```

Comment:
Adding a non-optional argument to a public function constitutes a breaking API change and needs to be annotated as such in the commit message for the change.

Reply:
I added the comment in a later commit and then squashed the commits. It _should_ be updated properly so that it gets caught by scripts, but I am not 100% sure I got it right. Please verify.